packages/react/src/FilteredActionList/FilteredActionList.examples.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type {Meta} from '@storybook/react-vite'
+import React, {useMemo} from 'react'
+import {FilteredActionList} from '../FilteredActionList'
+import classes from './FilteredActionList.stories.module.css'
+
+const meta: Meta = {
+  title: 'Components/FilteredActionList/Examples',
+  component: FilteredActionList,
+  parameters: {
+    controls: {
+      disable: true,
+    },
+  },
+}
+export default meta
+
+export function WithLongItems() {
```

Comment:
it's ok if we don't but it we want this to have VRT, we should add it to e2e/components/* (noticed we don't have one for FilteredActionList, might be a good time to add one)